skymp5-server/cpp/server_guest_lib/SweetPieBoundWeapon.cpp

```diff
@@ -0,0 +1,42 @@
+#include "SweetPieBoundWeapon.h"
```

Comment:
do not add a separate file for this OR
at least name files the same way (for example Not SwwetPie*** and PieScript, but
SweetPie******* and SweetPieScript)